3.Tests/UI/Features/Project/Actions.feature

```diff
@@ -19,3 +19,19 @@ Feature: Project Creation Above and Below
             And types "My New Project Name" on 'Edit Project Input'
             And clicks on 'Save Edit Project'
         Then the 'Project Button' <My New Project Name> should be displayed
+
+    @Regression @create.project.MyProject @delete.projects
+    Scenario: Change a project image
+        When the user opens the Project Context Menu on <MyProject> at 'Project Component'
+            And clicks on 'Shopping Bag Image'
+        Then the 'Project With Shopping Bag Image' should be displayed
+
+    @Regression @create.project.MyProject1 @create.project.MyProject2 @delete.projects @retrieve.projects
+    Scenario: Drag and drop above another project
+        When the user drags and drop 'MyProject2' 'Project Handle' above 'MyProject1' at 'Project Component'
+        Then the project 'MyProject2' should appear before 'MyProject1'
+
+    @Regression @create.project.MyProject1 @create.project.MyProject2 @delete.projects
+    Scenario: Drang and drop inside another project
```

Comment:
Please fix typo "Drang" -> "Drag"

Reply:
Thank you Sebastian, i didnt realice that haha my bad